guacamole/src/main/webapp/translations/ko.json

```
@@ -676,46 +676,46 @@
-        "FIELD_HEADER_LOGIN_SUCCESS_REGEX" : "Login success regular expression:",
-        "FIELD_HEADER_USERNAME"       : "Username:",
-        "FIELD_HEADER_USERNAME_REGEX" : "Username regular expression:",
-        "FIELD_HEADER_PASSWORD"       : "Password:",
-        "FIELD_HEADER_PASSWORD_REGEX" : "Password regular expression:",
-        "FIELD_HEADER_PORT"           : "Port:",
-        "FIELD_HEADER_READ_ONLY"      : "Read-only:",
-        "FIELD_HEADER_RECORDING_EXCLUDE_MOUSE"  : "Exclude mouse:",
-        "FIELD_HEADER_RECORDING_EXCLUDE_OUTPUT" : "Exclude graphics/streams:",
-        "FIELD_HEADER_RECORDING_INCLUDE_KEYS"   : "Include key events:",
-        "FIELD_HEADER_RECORDING_NAME" : "Recording name:",
-        "FIELD_HEADER_RECORDING_PATH" : "Recording path:",
-        "FIELD_HEADER_SCROLLBACK"     : "Maximum scrollback size:",
-        "FIELD_HEADER_TERMINAL_TYPE"   : "Terminal type:",
-        "FIELD_HEADER_TYPESCRIPT_NAME" : "Typescript name:",
-        "FIELD_HEADER_TYPESCRIPT_PATH" : "Typescript path:",
-        "FIELD_HEADER_WOL_BROADCAST_ADDR" : "Broadcast address for WoL packet:",
-        "FIELD_HEADER_WOL_MAC_ADDR"       : "MAC address of the remote host:",
-        "FIELD_HEADER_WOL_SEND_PACKET"    : "Send WoL packet:",
-        "FIELD_HEADER_WOL_WAIT_TIME"      : "Host boot wait time:",
+        "FIELD_HEADER_BACKSPACE"      : "<Backspace> 키가 보내는 문자:",
+        "FIELD_HEADER_COLOR_SCHEME"   : "색상 구성표:",
+        "FIELD_HEADER_CREATE_RECORDING_PATH" : "레코드 경로 자동 생성:",
+        "FIELD_HEADER_CREATE_TYPESCRIPT_PATH" : "Typescript 경로 자동 생성:",
+        "FIELD_HEADER_DISABLE_COPY"   : "터미널에서 복사 비활성화:",
+        "FIELD_HEADER_DISABLE_PASTE"  : "클라이언트에서 붙여넣기 비활성화:",
+        "FIELD_HEADER_FONT_NAME"      : "글꼴 이름:",
+        "FIELD_HEADER_FONT_SIZE"      : "글꼴 크기:",
+        "FIELD_HEADER_HOSTNAME"       : "호스트 이름:",
+        "FIELD_HEADER_LOGIN_FAILURE_REGEX" : "로그인 실패 정규식:",
+        "FIELD_HEADER_LOGIN_SUCCESS_REGEX" : "로그인 성공 정규식:",
+        "FIELD_HEADER_USERNAME"       : "유저 이름:",
+        "FIELD_HEADER_USERNAME_REGEX" : "유저 이름 정규식:",
+        "FIELD_HEADER_PASSWORD"       : "패스워드:",
+        "FIELD_HEADER_PASSWORD_REGEX" : "패스워드 정규식:",
+        "FIELD_HEADER_PORT"           : "포트:",
+        "FIELD_HEADER_READ_ONLY"      : "읽기 전용:",
+        "FIELD_HEADER_RECORDING_EXCLUDE_MOUSE"  : "마우스 제외:",
+        "FIELD_HEADER_RECORDING_EXCLUDE_OUTPUT" : "그래픽/스트림 제외:",
+        "FIELD_HEADER_RECORDING_INCLUDE_KEYS"   : "키 이벤트 포함:",
+        "FIELD_HEADER_RECORDING_NAME" : "레코딩 이름:",
+        "FIELD_HEADER_RECORDING_PATH" : "레코딩 경로:",
+        "FIELD_HEADER_SCROLLBACK"     : "스크롤백(scrollback) 최대 크기:",
+        "FIELD_HEADER_TERMINAL_TYPE"   : "터미널 타입:",
+        "FIELD_HEADER_TYPESCRIPT_NAME" : "Typescript 이름:",
+        "FIELD_HEADER_TYPESCRIPT_PATH" : "Typescript 경로:",
+        "FIELD_HEADER_WOL_BROADCAST_ADDR" : "Wol 패킷의 브로드캐스트 주소:",
+        "FIELD_HEADER_WOL_MAC_ADDR"       : "원격 호스트의 MAC 주소:",
```

```suggestion
        "FIELD_HEADER_WOL_MAC_ADDR"       : "원격 호스트 MAC 주소:",
```
'의'를 넣는게 번역체 스러운게 있어서 제외해봤습니다.